src/vmm/src/devices/virtio/net/persist.rs

```diff
@@ -37,14 +36,15 @@ pub struct NetConfigSpaceState {
 pub struct RxBufferState {
     // Number of iovecs we have parsed from the guest
     parsed_descriptor_chains_nr: u16,
-    deferred_descriptor: Option<ParsedDescriptorChain>,
+    // Number of deferred_descriptors
+    deferred_descriptors: u16,
 }
 
 impl RxBufferState {
     fn from_rx_buffers(rx_buffer: &RxBuffers) -> Self {
         RxBufferState {
             parsed_descriptor_chains_nr: rx_buffer.parsed_descriptors.len().try_into().unwrap(),
-            deferred_descriptor: rx_buffer.deferred_descriptor.clone(),
+            deferred_descriptors: rx_buffer.used_descriptors,
```

Comment:
added `used_bytes` as well to store/restore them.
But in general this would not have been a big problem to skip this field, as `used_bytes` are only used for rate limiter, and the worst thing which could have happened is if guest was throttled when snapshot was taken, it would be not be throttled for 1 packet after restore.